ocaml/typing/predef.ml

```diff
@@ -172,6 +172,7 @@ let mk_add_type add_type
      type_arity = 0;
      type_kind = kind;
      type_jkind = jkind;
+     type_jkind_annotation = None;
```

Comment:
Done. Leaving this for you to resolve in case you have comments on my approach for this. (Having the caller decide when to fill it in, like for `~jkind`.)